libs/blocks/carousel/carousel.css

```diff
@@ -358,6 +347,54 @@
 
 /* End Lightbox */
 
+/* Corrects backgournd images coming from section metadata */
+.carousel .carousel-slide.has-background {
+  position: relative;
+}
+
+/* Rest for nested blocks foreground */
+.carousel.container .carousel-slide .foreground:not(.icon-block .foreground) {
+  max-width: unset;
+  margin: unset;
+}
+
+.carousel.show-2 .carousel-slide .foreground,
+.carousel.show-3 .carousel-slide .foreground
+.carousel.show-4 .carousel-slide .foreground
+.carousel.show-5 .carousel-slide .foreground
+.carousel.show-6 .carousel-slide .foreground { width: auto; }
+
+.carousel.container .carousel-slide .container {
+  width: unset;
+}
+
+/* Zero out spacing token on carousel-slide */
+.carousel.container .carousel-slide.section.xs-spacing,
+.carousel.container .carousel-slide.section.s-spacing,
+.carousel.container .carousel-slide.section.m-spacing,
+.carousel.container .carousel-slide.section.l-spacing,
+.carousel.container .carousel-slide.section.xl-spacing,
+.carousel.container .carousel-slide.section.xxl-spacing { padding: 0; }
+
+/* Apply spacing token to block inside of slide */
+.carousel.container .carousel-slide.xs-spacing > div,
+.carousel.container .carousel-slide .xs-spacing { padding: var(--spacing-xs); }
+
+.carousel.container .carousel-slide.s-spacing > div,
+.carousel.container .carousel-slide .s-spacing { padding: var(--spacing-s); }
+
+.carousel.container .carousel-slide.m-spacing > div,
+.carousel.container .carousel-slide .m-spacing { padding: var(--spacing-m);  }
+
+.carousel.container .carousel-slide.l-spacing > div,
+.carousel.container .carousel-slide .l-spacing { padding: var(--spacing-l); }
+
+.carousel.container .carousel-slide.xl-spacing > div,
+.carousel.container .carousel-slide .xl-spacing { padding: var(--spacing-xl); }
+
+.carousel.container .carousel-slide.xxl-spacing > div
+.carousel.container .carousel-slide .xxl-spacing { padding: var(--spacing-xxl); }
+
```

Comment:
Do we want to include the `.section.xxxl-spacing` token in this as well? 
I see that it's not in the `typography.css` file either so maybe it's OK, or we need to add it to both to cover the bases.

Reply:
Good catch!

Hmm. the part I cleaned up in `typography.css` I think relates to the Text block. Do we need to update that to include `xxxl-spacing`? Can you or @Sartxi confirm?

It does makes sense to include in `carousel.css` since that is targeting `section-metadata` spacing tokens.

Reply:
Gotcha, Yea that `xxxl-spacing` isn't widely used but I feel like we should add it to `typography.css` as well just to cover all bases. It looks to have just been missed.

Reply:
The only thing i see is we should probably add this `xxxl-spacing` to `typography.css` for any block to potentially use. 
`div[class*='-up'] .con-block.has-bg.xxxl-spacing { padding: var(--spacing-xxxl); }`

Reply:
@ryanmparrish Added. Do you mind giving one more look over and adding the verified label is all looks good?